pkg/pmem-device-manager/pmd-ndctl.go

```diff
@@ -33,10 +32,6 @@ func NewPmemDeviceManagerNdctl() (PmemDeviceManager, error) {
 	ndctlMutex.Lock()
```

Comment:
Now in this function there is nothing to protect with mutex any more?

Reply:
True, can be removed.

Reply:
Done.